sleap/info/write_tracking_h5.py

```diff
@@ -120,7 +120,11 @@ def get_occupancy_and_points_matrices(
     instance_scores = np.full((frame_count, track_count), np.nan, dtype=float)
     tracking_scores = np.full((frame_count, track_count), np.nan, dtype=float)
 
-    for lf, inst in [(lf, inst) for lf in labeled_frames for inst in lf.instances]:
+    # TODO: Revisit with LabeledFrames.unused_predictions() & instances_to_show()
+    lfs_instances = [
+        (lf, inst) for lf in labeled_frames for inst in lf.instances_to_show
+    ]
```

Comment:
It seems `instances_to_show` is turning out to be very useful here and in #649, but there was some [discrepancy](https://github.com/talmolab/sleap/pull/694#discussion_r840869742) with `unused_predictions` and how it was implemented. Should we revisit these two methods now?